src/appleseed.shaders/include/appleseed/color/as_color_transforms.h

```diff
@@ -1045,36 +1045,44 @@ color transform_CIELCh_uv_to_linear_RGB(
 }
 
 //
-// CIELAB, CIELUV, have values in range L [0,100], a,b [-100,100]
-// and u,v [-100,100].
+// CIELAB, CIELUV, have values in range L [0,100], a,b [-128,128]
+// and u,v ([-134,220],[-140,122])
```

Comment:
Kind of weird no? What's the rationale?

Reply:
I wholeheartedly agree, but that seems to be the case. It is based on u, v coordinates, http://scc.ustc.edu.cn/zlsc/sugon/intel/ipp/ipp_manual/IPPI/ippi_ch6/ch6_color_models.htm
The entire rationale for this space eludes me so far.